spec/integration/cds_importer/entity_mapper_spec.rb

```diff
@@ -26,20 +26,51 @@
 
   describe '#import' do
     before do
-      stub_const(
```

Comment:
This isn't needed to get the tests to pass and misdirected me into thinking I need to change something I didn't